rpc/internal/test_results_service.go

```diff
@@ -83,16 +84,63 @@ func (s *testResultsService) AddTestResults(ctx context.Context, results *TestRe
 	}
 
 	if !record.Info.HistoricalDataDisabled {
-		for _, res := range exportedResults {
-			// TODO: Remove this job with EVG-14440.
-			j := units.NewHistoricalTestDataJob(s.env, record.Info, res)
-			j.Run(ctx)
-			grip.Error(message.WrapError(j.Error(), message.Fields{
-				"message":     "failed to run historical test data job",
-				"info":        record.Info,
-				"test_result": res,
-			}))
-		}
+		go func() {
```

Comment:
Is there a concern that this will start too many goroutines since it has to run often? Assuming `Update` is about as performant as the other queries in `AddTestResults` like `Append`, is it necessary to have a separate goroutine for it?

Reply:
There will only be one goroutine per call to `AddTestResults`, previously it was blowing up in the queue because it was a job per test result (whereas this is one goroutine for all the test results). I think it should be fine. Since it does have to iterate through potentially many test results, though, I would rather keep it in a goroutine so we do not slow down requests.